sdk/resourcemanager/azure-resourcemanager-compute/src/main/java/com/azure/resourcemanager/compute/implementation/DiskImpl.java

```diff
@@ -137,6 +137,11 @@ public Mono<Void> revokeAccessAsync() {
         return this.manager().serviceClient().getDisks().revokeAccessAsync(this.resourceGroupName(), this.name());
     }
 
+    @Override
+    public boolean supportsHibernation() {
```

Comment:
not sure if `isHibernationSupported` be better? hard to tell by me.

Reply:
https://github.com/Azure/azure-sdk-for-java/blob/d1fbfcb0804ed2eb848ca79a00128d4d708016eb/sdk/resourcemanager/azure-resourcemanager-compute/src/main/java/com/azure/resourcemanager/compute/models/Disk.java#L31
Here we also use `isAttachedToVirtualMachine`. Maybe `isHibernationSupported ` is better?

Reply:
Yes, `isHibernationSupported` is better to align with current mgmt pattern for boolean, thanks!